content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -71,79 +71,78 @@ <h2 id="why-another-data-channel">Why another data channel?</h2>
 
 <p>We have <a href="http://www.html5rocks.com/en/tutorials/websockets/basics/">WebSocket</a>, <a href="http://www.html5rocks.com/en/tutorials/file/xhr2/">AJAX</a> and <a href="http://www.html5rocks.com/en/tutorials/eventsource/basics/">Server Sent Events</a>. Why do we need another communication channel? WebSocket is bidirectional, but all these technologies are designed for communication to or from a server.</p>
 
-<p>RTCDataChannel takes a different approach:</p>
+<p><code>RTCDataChannel</code> takes a different approach:</p>
 
 <ul>
-  <li>It works with the RTCPeerConnection API, which enables peer to peer connectivity. This can result in lower latency: no intermediary server, fewer 'hops'.</li>
-  <li>RTCDataChannel uses <a href="https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol#Features">Stream Control Transmission Protocol</a> (SCTP), allowing configurable delivery semantics: out-of-order delivery and retransmit configuration.</li>
+ <li>It works with the <code>RTCPeerConnection</code> API, which enables peer-to-peer connectivity. This can result in lower latency&mdash;no intermediary server and fewer 'hops'.</li>
+ <li><code>RTCDataChannel</code> uses <a href="https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol#Features">Stream Control Transmission Protocol</a> (SCTP), allowing configurable delivery semantics&mdash;out-of-order delivery and retransmit configuration.</li>
 </ul>
 
-<p>RTCDataChannel is available now, with SCTP support, in Chrome, Opera and Firefox for desktop and Android.</p>
+<p><code>RTCDataChannel</code> is available now with SCTP support on desktop and Android in Google Chrome, Opera, and Firefox.</p>
 
-<h2>A caveat: signaling, STUN and TURN</h2>
+<h2>A caveat: Signaling, STUN, and TURN</h2>
 
-<p>Although WebRTC enables peer-to-peer communication, it still needs servers:</p>
+<p>Although WebRTC enables peer-to-peer communication, it still needs servers for:</p>
 
 <ul>
-  <li><strong>For signaling:</strong> to enable the exchange of media and network metadata to bootstrap a peer connection.</li>
-  <li><strong>To cope with NATs and firewalls</strong>: by using the ICE framework to establish the best possible network path between peers, by working with STUN servers (to ascertain a publicly accessible IP and port for each peer) and TURN servers (if direct connection fails and data relaying is required).</li>
+  <li>Signaling&mdash;the exchange of media and network metadata to bootstrap a peer connection</li>
+  <li>Coping with network address translators (NATs) and firewalls</li>
```

Comment:
I'd be inclined to leave in the rest of this sentence, or including the information some other ways. 

Maybe these items could become the bullet points, something like this:

```
WebRTC enables peer-to-peer communication, but it still needs servers for <strong>signaling</strong>, to exchange media and network metadata to bootstrap a peer connection. 

WebRTC copes with NATs and firewalls by using: 
<ul>
  <li><strong>The ICE framework</strong> to establish the best possible network path between peers.</li>
  <li><strong>STUN servers</strong> to ascertain a publicly accessible IP and port for each peer</li>
  <li><strong>TURN servers</strong> if direct connection fails and data relaying is required.</li>	 
```

Reply:
This copy is great! Thanks!

I also added links, but please check them.